geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSetTest.java

```diff
@@ -193,9 +197,10 @@ public void should_calculateSize_equalToROS_withNoMembers() {
   }
 
   @Test
+  @Ignore("Sizing tests are known to be flaky/incorrect and will be fixed as part of GEODE-9279")
   public void should_calculateSize_equalToROS_withSingleMember() {
-    HashSet<ByteArrayWrapper> members = new HashSet<>();
-    members.add(new ByteArrayWrapper("value".getBytes()));
+    Set<byte[]> members = new HashSet<>();
```

Comment:
I guess these tests are technically correct, but it still seems weird to use a HashSet<byte[]>() here. That will use the Object.hashCode() of the byte[] array. Maybe make this a ArrayList<> instead to make it clear we don't really care about set semantics on `members` here? Or use Object2ObjectOpenCustomHashMap instead.